addons/spreadsheet/static/src/chart/plugins/chart_odoo_menu_plugin.js

```diff
@@ -16,7 +16,27 @@ export default class ChartOdooMenuPlugin extends CorePlugin {
     handle(cmd) {
         switch (cmd.type) {
             case "LINK_ODOO_MENU_TO_CHART":
-                this.history.update("odooMenuReference", cmd.chartId, cmd.odooMenuId);
+                {
+                    this.history.update("odooMenuReference", cmd.chartId, cmd.odooMenuId);
```

Comment:
We can totally drop `this.odooMenuReference` and use the information in the chart definition right ? So we won't have to handle `CREATE/DELETE_CHART`.

I'd also think that import/export the `odooMenuId` should be done by the chart when it export/import its definition, not by this plugin.